app/src/ui/app.tsx

```diff
@@ -583,6 +583,27 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.setAppMenuState(menu => menu.withReset())
   }
 
+  private onTitlebarDoubleClick() {
+    if (__DARWIN__) {
+      const actionOnDoubleClick = remote.systemPreferences.getUserDefault('AppleActionOnDoubleClick', 'string')
+      const mainWindow = remote.getCurrentWindow()
+
+      if (mainWindow) {
+        switch (actionOnDoubleClick) {
+          case 'Maximize':
+          default:
```

Comment:
There isn't, but Apple seems to change this kind of interface all the time, so I erred on the side of *some* behavior being better than nothing just in case - totally fine to make the default a noop instead, though